code/datums/stars.dm

```diff
@@ -141,11 +140,22 @@
 		map_names += "empty_space.dmm"
 	
 	if(prob(50))
-		// For now, all planets are LAVALAND
-		map_names += "lavaland.dmm"
-		planet_type = "Lava Planet"
-		surface_turf_type = /turf/open/floor/plating/asteroid/basalt/lava_land_surface
-		surface_area_type = /area/lavaland/surface/outdoors
+		switch(rand(1, 100))
```

Comment:
There will be more types, and this allows me better control than the above, since you sort of have to do one of those things:

```
if(prob(25))
    // a planet
else if(prob(33))
    // Another planet
else if(prob(50))
    // Yet another planet
else
    // What do you think?
```